app/controllers/admin/roles_controller.rb

```diff
@@ -103,6 +103,10 @@ def toggle_user
 
     protected
 
+    def user_params
+      params.require(:user).permit(:email, :state)
+    end
+
```

Comment:
I manually undid the changes in this file that caused this. It should be fixed now!